### DISCUSSION TITLE
Config Not Working Properly

### DISCUSSION THREAD

**Discussion by yang1782** [swar/Swar-Chia-Plot-Manager#1002] [Q&A]
I have a AMD Ryzen 5950x with two 2TB NVme and 64GB of RAM. It appears my plot manager is not following the config I have setup for it. As you can see, even though I have each Nvme set to run a max concurrent of 10 plots and max for phase 1 of 3, there's something that's limiting me from reaching those max. Please help. Here's my config for each NVme:

  - name: SamPlus970-1
    max_plots: 999
    farmer_public_key:
    pool_public_key:
    temporary_directory: F:\Plotter
    temporary2_directory:
    destination_directory: 
     - D:\Plots
    size: 32
    bitfield: true
    threads: 4
    buckets: 128
    memory_buffer: 6750
    max_concurrent: 10
    max_concurrent_with_start_early: 10
    initial_delay_minutes: 0
    stagger_minutes: 20
    max_for_phase_1: 3
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false
    exclude_final_directory: false
    skip_full_destinations: true
    unix_process_priority: 10
    windows_process_priority: 32
    enable_cpu_affinity: false
    cpu_affinity: [ 0, 1, 2, 3, 4, 5 ]

![Progress](https://user-images.githubusercontent.com/84889848/120078204-e32bc280-c073-11eb-8872-d4c24009854f.PNG)

**Comment by Andresit0** [comment]
2TB NVme allow only 7 plots at the same time (256 GB of temporary space for each plot). Then, max_concurrent: 7
